manifests/f/Firezone/Client/GUI/1.4.14/Firezone.Client.GUI.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: Firezone.Client.GUI
+PackageVersion: 1.4.14
+PackageLocale: en-US
+Publisher: Firezone
+PackageName: Firezone GUI Client
```

```suggestion
PackageName: Firezone
```
This must match Control Panel.